diff --git a/dist/node/chunks/config.js b/dist/node/chunks/config.js
index 782dc4d707f0009436224f3577f4fc46f3c68378..c60deaa6e8c09ca7e82af97b12826f8a63ca3172 100644
--- a/dist/node/chunks/config.js
+++ b/dist/node/chunks/config.js
@@ -32443,7 +32443,8 @@ const makeScssWorker = (environment, resolvers, _maxWorkers) => {
 	let compilerPromise;
 	return {
 		async run(sassPath, data, options$1) {
-			const sass = (await import(pathToFileURL(sassPath).href)).default;
+			const sassImport = await import(pathToFileURL(sassPath).href);
+			const sass = sassImport.default ?? sassImport;
 			compilerPromise ??= sass.initAsyncCompiler();
 			const compiler = await compilerPromise;
 			const sassOptions = { ...options$1 };
